solutions/swb-app/src/sshKeys/createSshKeyRequest.ts

```diff
@@ -2,13 +2,13 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-import { z } from 'zod';
+import { z } from '@aws/workbench-core-base';
 
 // eslint-disable-next-line @rushstack/typedef-var
 export const CreateSshKeyRequestParser = z
   .object({
-    projectId: z.string(),
-    userId: z.string()
+    projectId: z.string().projId().required(),
+    userId: z.string().required()
```

Comment:
unfortunately there should be an interface in ZodString on userId(), and these work should have been covered in the generic parser ticket.....

Reply:
got it. I figured someone else had that ticket but I guess #scope-creep and I will add it here!